package.json

```
@@ -2,25 +2,43 @@
   "name": "eslint-visitor-keys",
   "version": "2.1.0",
   "description": "Constants and utilities about visitor keys to traverse AST.",
-  "main": "lib/index.js",
+  "type": "module",
+  "main": "dist/eslint-visitor-keys.cjs",
+  "exports": {
+    ".": [
+      {
+        "import": "./lib/index.js",
+        "require": "./dist/eslint-visitor-keys.cjs",
+        "default": "./dist/eslint-visitor-keys.cjs"
+      },
+      "./dist/eslint-visitor-keys.cjs"
+    ],
+    "./package.json": "./package.json"
+  },
   "files": [
+    "dist",
     "lib"
   ],
   "engines": {
-    "node": ">=10"
+    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
   },
   "devDependencies": {
-    "eslint": "^4.7.2",
-    "eslint-config-eslint": "^4.0.0",
+    "c8": "^7.7.3",
+    "eslint": "^7.29.0",
+    "eslint-config-eslint": "^7.0.0",
+    "eslint-plugin-jsdoc": "^35.4.0",
+    "eslint-plugin-node": "^11.1.0",
     "eslint-release": "^3.1.2",
-    "mocha": "^3.5.3",
-    "nyc": "^11.2.1",
-    "opener": "^1.4.3"
+    "mocha": "^9.0.1",
+    "opener": "^1.5.2",
+    "rollup": "^2.52.1"
   },
   "scripts": {
+    "prepare": "npm run build",
+    "build": "rollup -c",
     "lint": "eslint lib tests/lib",
```

```suggestion
    "lint": "eslint .",
```

to lint `rollup.config.js` and `commonjs.cjs`, too.